pkg/controller/disruption/disruption_test.go

```diff
@@ -1584,9 +1587,3 @@ func verifyEventEmitted(t *testing.T, dc *disruptionController, expectedEvent st
 		}
 	}
 }
-
-// TestMain adds klog flags to make debugging tests easier.
-func TestMain(m *testing.M) {
-	klog.InitFlags(flag.CommandLine)
-	os.Exit(m.Run())
-}
```

Comment:
`ktesting` package `Init()` function has set `-v` flag to make debugging tests easier, so we can remove the `klog.InitFlags` function.